### PR TITLE
binary search added

### PR DESCRIPTION
**Pull request by rahuTi**
binary search added

### CODE REVIEW

File: c1.txt [hunk 1]
```diff
@@ -2,4 +2,51 @@ this is my code...
...
+        int x = 10;
+        int result = ob.binarySearch(arr, x);
+        if (result == -1)
+            System.out.println(
+                "Element is not present in array");
+        else
+            System.out.println("Element is present at "
+                               + "index " + result);
+    }
+}
```
Comment (rahuTi) [inline c1.txt:52]: /explain

File: c1.txt [hunk 2]
```diff
@@ -2,4 +2,51 @@ this is my code...
...
+    // Driver code
+    public static void main(String args[])
+    {
+        BinarySearch ob = new BinarySearch();
+        int arr[] = { 2, 3, 4, 10, 40 };
+        int n = arr.length;
+        int x = 10;
+        int result = ob.binarySearch(arr, x);
+        if (result == -1)
+            System.out.println(
```
Comment (rahuTi) [inline c1.txt:46]: what language is used in above code and what algo is used?

File: c1.txt [hunk 3]
```diff
@@ -2,4 +2,51 @@ this is my code...
...
+ 
+    // Driver code
+    public static void main(String args[])
+    {
+        BinarySearch ob = new BinarySearch();
+        int arr[] = { 2, 3, 4, 10, 40 };
+        int n = arr.length;
+        int x = 10;
+        int result = ob.binarySearch(arr, x);
+        if (result == -1)
```
Comment (Rahu09) [inline c1.txt:45]: /explain

Comment (Rahu09) [inline c1.txt:45]: /explain

**Pull request closed without merging**